onset_maproom/maproom.py

```diff
@@ -462,9 +467,9 @@ def onset_tile(tz, tx, ty):
     y_max = pingrid.tile_bottom_mercator(ty, tz)
     y_min = pingrid.tile_bottom_mercator(ty + 1, tz)
 
-    precip_tile = rr_mrg.precip.sel(
-        X=slice(x_min, x_max),
-        Y=slice(y_min, y_max),
+    precip_tile = rr_mrg.precip.interp(
+        X=np.arange(x_min - x_min % .25, x_max + RESOLUTION - x_max % RESOLUTION, RESOLUTION),
+        Y=np.arange(y_min - y_min % .25, y_max + RESOLUTION - y_max % RESOLUTION), RESOLUTION),
```

Comment:
I would need to be a bit more acquainted on the tiling process to comment on that. Let me know if you'd like me to and I'll do my homework

Reply:
Igor wrote the tiling code and I reviewed it, so I don't feel I need you to dig into it, although it might be helpful in the future to know about it. I can talk you through it if you want.

On the other hand, you probably know more than me about the various options for interpolation/regridding. I just added a comment about that, please refresh the PR and read that.

Reply:
@remicousin apparently you're not reviewing the latest version of this branch. The interpolation now happens in enactstozarr.py, not here.

I think updating will also fix the colorbar problem you mentioned.

Reply:
Oh, you're probably reviewing commit-by-commit as I recommended. :-/

Reply:
yes